cli/script/command-executor.ts

```diff
@@ -659,17 +786,45 @@ function formatDate(unixOffset: number): string {
     }
 }
 
+function getAppDisplayName(app: App): string {
+    return app.isOwner ? app.name : getOwnerEmail(app.collaborators) + "/" + app.name;
+}
+
 function printAppList(format: string, apps: App[], deploymentLists: string[][]): void {
     if (format === "json") {
         var dataSource: any[] = apps.map((app: App, index: number) => {
-            return { "name": app.name, "deployments": deploymentLists[index] };
+
+            return { "name": getAppDisplayName(app), "deployments": deploymentLists[index] };
         });
+
         printJson(dataSource);
     } else if (format === "table") {
         var headers = ["Name", "Deployments"];
         printTable(headers, (dataSource: any[]): void => {
             apps.forEach((app: App, index: number): void => {
-                var row = [app.name, wordwrap(50)(deploymentLists[index].join(", "))];
+                var row = [getAppDisplayName(app), wordwrap(50)(deploymentLists[index].join(", "))];
+                dataSource.push(row);
+            });
+        });
+    }
+}
+
+function getCollaboratorDisplayName(collaborator: Collaborator): string {
+    return (collaborator.permission === "Owner") ? collaborator.email + chalk.magenta(" (Owner)") : collaborator.email;
+}
+
+function printCollaboratorsList(format: string, collaborators: Collaborator[]): void {
+    if (format === "json") {
+        var dataSource: any[] = collaborators.map((collaborator: Collaborator) => {
+            return { "email": getCollaboratorDisplayName(collaborator) };
```

Comment:
In the case of the JSON format, we probably want the owner/collaborator distinction provided in a structure way, instead of relying on a display name. Should we just return the "raw" collaborators list?